.github/workflows/update_documentation.yml

```
@@ -87,7 +87,7 @@ jobs:
         cd t8code-website/content
         rm -rf doc/dev/*
         mkdir -p doc/dev
-        cp -r ../../build_debug/doxygen/html/* doc/dev/
+        cp -r ../../doc/Doxyfile/* doc/dev/
```

```suggestion
        cp -r ../../build_debug/doc/html/* doc/dev/
```
You can find out the new path by checking out and updating your main branch and then building t8code from scratch, starting with I think the `git submodule update` command. Git will then update the SC and p4est library and adapt to the new documentation path. After configuring, you can use `make doxygen` to generate the documentation. In the old version, the html documentation was written to `<build_folder>/doxygen/html/`. With the new patch, this changes to `<build_folder>/doc/html/`.